libinput: 1.15.6 -> 1.16.1

r-ryantm opened the pull request:
Automatic update generated by [nixpkgs-update](https://github.com/ryantm/nixpkgs-update) tools. This update was made based on information from https://repology.org/metapackage/libinput/versions.


meta.description for libinput is: "Handles input devices in Wayland compositors and provides a generic X.Org input driver"

meta.homepage for libinput is: "http://www.freedesktop.org/wiki/Software/libinput"

meta.changelog for libinput is: ""


###### Updates performed
- Version update

###### To inspect upstream changes





###### Impact

<details>
<summary>
<b>Checks done</b> (click to expand)
</summary>

---

- built on NixOS
- The tests defined in `passthru.tests`, if any, passed

- 0 of 0 passed binary check by having a zero exit code.
- 0 of 0 passed binary check by having the new version present in output.
- found 1.16.0 with grep in /nix/store/q42v711342sjks1r5b9380l5dgb9f0jh-libinput-1.16.0-bin
- directory tree listing: https://gist.github.com/99b37893fcc1837eb8eb4b1344f64e7e

- du listing: https://gist.github.com/42c5e39c692cf777b13004e2e3ec758e


---

</details>
<details>
<summary>
<b>Rebuild report</b> (if merged into master) (click to expand)
</summary>

```
4975 total rebuild path(s)

1749 package rebuild(s)

1749 x86_64-linux rebuild(s)
1619 i686-linux rebuild(s)
6 x86_64-darwin rebuild(s)
1601 aarch64-linux rebuild(s)


First fifty rebuilds by attrpath
AusweisApp2
adapta-gtk-theme
adwaita-qt
aj-snapshot
akonadi
akregator
albert
amarok
amarok-kf5
ams
android-file-transfer
anki
ankisyncd
antimicroX
antimony
ao
apitrace
apmplanner2
appcsxcad
appstream-qt
aqemu
ark
arx-libertatis
asymptote
audacious
audaciousQt5
avidemux
awesomebump
bambootracker
barrier
bcompare
beamerpresenter
beets
bibletime
bino3d
birdtray
bitcoin
bitcoin-abc
bitcoin-classic
bitcoin-gold
bitcoin-unlimited
bluedevil
bomber
bomi
boomerang
bovo
breeze-grub
breeze-gtk
breeze-icons
breeze-plymouth
```

</details>

<details>
<summary>
<b>Instructions to test this update</b> (click to expand)
</summary>

---

Either **download from Cachix**:
```
nix-store -r /nix/store/q42v711342sjks1r5b9380l5dgb9f0jh-libinput-1.16.0-bin \
  --option binary-caches 'https://cache.nixos.org/ https://r-ryantm.cachix.org/' \
  --option trusted-public-keys '
  r-ryantm.cachix.org-1:gkUbLkouDAyvBdpBX0JOdIiD2/DP1ldF3Z3Y6Gqcc4c=
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  '
```
(r-ryantm's Cachix cache is only trusted for this store-path realization.)
For the Cachix download to work, your user must be in the `trusted-users` list or you can use `sudo` since root is effectively trusted.

Or, **build yourself**:
```
nix-build -A libinput https://github.com/r-ryantm/nixpkgs/archive/a901a07d7471156591ca6a8bdb0c4ea65e1133a1.tar.gz
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/q42v711342sjks1r5b9380l5dgb9f0jh-libinput-1.16.0-bin
ls -la /nix/store/q42v711342sjks1r5b9380l5dgb9f0jh-libinput-1.16.0-bin/bin
```

---

</details>
<br/>



### Pre-merge build results

NixPkgs review skipped

---

###### Maintainer pings

cc @codyopel for testing.

jtojnar commented:
Looks like there was a regression: https://lists.freedesktop.org/archives/wayland-devel/2020-August/041590.html

jtojnar committed:
libinput: 1.15.6 -> 1.16.1

https://lists.freedesktop.org/archives/wayland-devel/2020-August/041578.html
https://lists.freedesktop.org/archives/wayland-devel/2020-August/041590.html

jtojnar reviewed (approved):
Briefly looked at the [diff](https://gitlab.freedesktop.org/libinput/libinput/-/compare/1.15.6...1.16.1), did not see any issues.
